app/views/admin/reports/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<h1>Reports</h1>
+<p id="notice"><%= notice %></p>
+
+<h3>Orders in last 5 days</h3>
+<%= form_for(:report) do |form| %>
+  <div class="field">
+    <%= form.label :from_date %>
+    <%= form.date_field :from_date, id: :from_date %>
+  </div>
+
+  <div class="field">
+    <%= form.label :to_date %>
+    <%= form.date_field :to_date, id: :to_date %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit %>
```

Comment:
give a name say `filter` to submit button
also, lets move filtering logic to a partial for better code management